data.sql

```diff
@@ -1,5 +1,10 @@
 /* Populate database with sample data. */
 
-INSERT INTO animals (name) VALUES ('Luna');
-INSERT INTO animals (name) VALUES ('Daisy');
-INSERT INTO animals (name) VALUES ('Charlie');
+INSERT INTO animals (id, name, date_of_birth, escape_attempts,neutered, weight_kg) 
```

Comment:
- [ OPTIONAL]
Take note that if you make `id` column to `autoincrement`, then you don't need to add `id` to the `INSERT` statement here again.